src/tv2-common/helpers/graphics/name.ts

```diff
@@ -34,7 +34,7 @@ export function GetFullGraphicTemplateNameFromCue(
 }
 
 export function GetFullGrafikTemplateName(config: TV2BlueprintConfig, iNewsTempalateName: string): string {
-	const template = config.showStyle.GFXTemplates.find(templ =>
+	const template = config.showStyle.GfxTemplates.find(templ =>
 		templ.INewsName ? templ.INewsName.toString().toUpperCase() === iNewsTempalateName.toUpperCase() : false
```

Comment:
I know this wasn't you but could we change it from `INewsTempalateName` to `INewsTemplateName` here?